Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -584,19 +586,22 @@ public void Init(IEOSCoroutineOwner coroutineOwner, string configFileName)
                         LoggingInterface.SetCallback(SimplePrintCallback);
                         hasSetLoggingCallback = true;
                     }
-#if UNITY_EDITOR
-                    SetLogLevel(LogCategory.AllCategories, LogLevel.VeryVerbose);
-#else
-                    SetLogLevel(LogCategory.AllCategories, LogLevel.Warning);
-#endif
+
+                    for (int logCategoryIndex = 0; logCategoryIndex < logLevelList.Count; logCategoryIndex++)
```

Comment:
I feel like since this shows up in three places, we should make a private function that its purpose is to do this code.